### DISCUSSION TITLE
Форма ввода с двумя и более полями

### DISCUSSION THREAD

**Discussion by an220022** [totumonline/totum-ru-issues-and-discussions#57] [Как это сделать?] [answered]
Я решил сделать форму ввода данных для редактирования полей строки простой таблицы, вызываемую из другой таблицы. Для этого создал временную таблицу, предоставил к ней доступы и скрыл из дерева. Затем использовал вызов формы из кода действия функцией linkToPanel с передачей через параметр 'field' определенных значений в поля панели. Требовалось запретить редактирование или совсем не  показывать эти значения полей панели. После того, как я снял галку "Вводится при добавлении в WEB" перестали подставляться значения в поля панели. Как сделать запрет на редактирование полей панели для пользователей или не показывать эти поля в панели?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Когда открывается linkToPanel, то параметры `field: 'name' = VALUE` передаются в адресной строке. Причем они передаются открытым способом. Поэтому эта функция невзломостойкая и `field` здесь для удобства. Именно поэтому отключения доступа при добавлении выключает подстановку. Если требуется защищенный вариант — то через временную таблицу.

Если у вас это место в решении не критично и специально его никто ломать не будет то можно просто скрыть это поле для панели. В форматировании поля:

```
f1=: setFormat(hide: "panel" = true)
```
